index.js

```diff
@@ -1,15 +1,32 @@
-var FluxConstant = function FluxConstant(name) {
+var FluxConstant = function FluxConstant(name, id) {
 
-    this.name = name;
+    this.id = String(id || (FluxConstant.lastId += 1));
```

Comment:
just saw your comment, that makes sense. If we serialize + deserialize the state of `lastId` as well, we are fine.